### PR TITLE
Clean up language/consistency in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kaicataldo**
Am interested in contributing to Babel and was reading through the CONTRIBUTING.md file when I noticed some inconsistencies/unclear explanations. Tried to clean them up a bit to hopefully help out the next person.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,16 +17,15 @@
...
 ## Not sure where to start?
 
 - If you aren't just making a documentation change, you'll probably want to learn a bit about a few topics.
  - [ASTs](https://en.wikipedia.org/wiki/Abstract_syntax_tree) (Abstract Syntax Tree): Our current [spec](https://github.com/babel/babel/tree/master/doc/ast) is a bit different from [ESTree](https://github.com/estree/estree).
- - [`/doc`](/doc) for notes on babel internals
- - Check out [the babel plugin handbook](https://github.com/thejameskyle/babel-handbook/blob/master/translations/en/plugin-handbook.md#babel-plugin-handbook). Core babel plugins are written the same way as any other plugin!
- - Checkout [AST Explorer](http://astexplorer.net/#/scUfOmVOG5) to learn more about ASTs or making your own plugin live
+ - This repository's [`/doc`](/doc) directory for notes on Babel's internals
+ - Check out [the Babel Plugin Handbook](https://github.com/thejameskyle/babel-handbook/blob/master/translations/en/plugin-handbook.md#babel-plugin-handbook) - core plugins are written the same way as any other plugin!
+ - Check out [AST Explorer](http://astexplorer.net/#/scUfOmVOG5) to learn more about ASTs or make your own plugin in the browser
```
Comment (kaicataldo) [inline CONTRIBUTING.md]: "Check out" is used in the previous bullet point

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -17,16 +17,15 @@
...
+contributing, please read our [code of conduct](https://github.com/babel/babel/blob/master/CODE_OF_CONDUCT.md).
 
 ## Not sure where to start?
 
 - If you aren't just making a documentation change, you'll probably want to learn a bit about a few topics.
  - [ASTs](https://en.wikipedia.org/wiki/Abstract_syntax_tree) (Abstract Syntax Tree): Our current [spec](https://github.com/babel/babel/tree/master/doc/ast) is a bit different from [ESTree](https://github.com/estree/estree).
- - [`/doc`](/doc) for notes on babel internals
- - Check out [the babel plugin handbook](https://github.com/thejameskyle/babel-handbook/blob/master/translations/en/plugin-handbook.md#babel-plugin-handbook). Core babel plugins are written the same way as any other plugin!
- - Checkout [AST Explorer](http://astexplorer.net/#/scUfOmVOG5) to learn more about ASTs or making your own plugin live
+ - This repository's [`/doc`](/doc) directory for notes on Babel's internals
```
Comment (kaicataldo) [inline CONTRIBUTING.md]: Babel is sometimes capitalized and sometimes not

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -92,25 +91,24 @@ $ make test-cov
...
 
 - There is an `index.js` file. It imports our [test helper](/packages/babel-helper-plugin-test-runner). (You don't have to worry about this).
 - There can be multiple folders under [`/fixtures`](/packages/babel-plugin-transform-exponentiation-operator/test/fixtures)
-   - There is an [`options.json`](/packages/babel-plugin-transform-exponentiation-operator/test/fixtures/exponentian-operator/options.json) is basically a `.babelrc` file to pass in the plugins and settings you need for your tests.
+   - There is an [`options.json`](/packages/babel-plugin-transform-exponentiation-operator/test/fixtures/exponentian-operator/options.json) file whose function is similar to a `.babelrc` file, allowing you to pass in the plugins and settings you need for your tests.
    - For this test, we only need the relevant plugin, so it's just `{ "plugins": ["transform-exponentiation-operator"] }`.
-   - If necessary, you can specify a different `options.json` for each sub folder if you need different options.
+   - If necessary, you can have an `options.json` with different options in each subfolder.
```
Comment (kaicataldo) [inline CONTRIBUTING.md]: Consist spelling of "subfolder"

**Comment by hzoo** [comment]
Awesome thanks for this first contribution - should make it easier to read through for new people :smile:

**Pull request merged**